README.md

```diff
@@ -177,9 +177,11 @@ If the `showClearDates` prop is set to true, an `x` shows up in the input box th
   showClearDates: PropTypes.bool,
 ```
 
-If the `showCalendarIcon` prop is set to true, a calendar icon is displayed at the start of the input box. Interaction is the same as focusing the start date.
+If the `showInputIcon` prop is set to true, a calendar icon is displayed at the start of the input box. Interaction is the same as focusing the start date.
+Optionally, it will display a React node set to `props.customInputIcon`
 ```
-  showCalendarIcon: PropTypes.bool
+  showInputIcon: PropTypes.bool,
+  customInputIcon: PropTypes.node
```

Comment:
ah, i didn't realize there'd be an icon included by default. Might it be simpler to go with a single `inputIcon` prop, and then export the default icon so that users can pass that in if they want it?